3.evaluate_model/scripts/nbconverted/calculate_silhouette_metrics.py

```diff
@@ -0,0 +1,133 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ## Calculate Silhouette Scores
+# 
+# - per phenotype
+# - per feature space
+
+# In[1]:
+
+
+import pathlib
+import numpy as np
+import pandas as pd
+
+from sklearn.decomposition import PCA
+from sklearn.metrics import silhouette_score, silhouette_samples
+
+import sys
+
+sys.path.append("../utils")
+from split_utils import get_features_data
+
+
+# In[2]:
+
+
+np.random.seed(1234)
+
+# For consistent Silhouette input space dimensionality
+n_pca_components = 50
+
+
+# In[3]:
+
+
+eval_path = pathlib.Path("evaluations")
+
+output_silhouette_results = pathlib.Path(
+    eval_path, "silhouette_score_results.tsv"
+)
+output_silhouette_samples_results = pathlib.Path(
+    eval_path, "silhouette_score_results_per_sample.tsv"
+)
```

Comment:
Recommend making these compressed TSVs for saving space plus that might be the standard convention in this repo.

Reply:
they are super tiny, will stick with tsv so they can be rendered on github